src/integration_tests/test_provisioning.py

```diff
@@ -39,6 +40,10 @@ def test_create(self):
             lifecycle.start(stackname)
 
             cfn.cmd(stackname, "ls -l /", username=BOOTSTRAP_USER, concurrency='parallel')
+            cfn.download_file(stackname, "/bin/ls", "ls", use_bootstrap_user="true")
+            self.assertTrue(os.path.exists("./ls"))
+            cfn.download_file(stackname, "/bin/less", "venv/bin/", use_bootstrap_user="true")
+            self.assertTrue(os.path.exists("./venv/bin/less"))
 
```

Comment:
add a test case to download to a destination that cannot be created that errors out